addons/test_mass_mailing/tests/common.py

```diff
@@ -138,3 +138,21 @@ def setUpClass(cls):
             phone_validation.phone_format(partner.mobile, partner.country_id.code, partner.country_id.phone_code, force_format='E164')
             for partner in partners
         ]
+
+    @classmethod
+    def _get_sms_test_records(cls, mobile_numbers):
+        """ Helper to create data. Currently simple, to be improved. """
+        country_be_id = cls.env.ref('base.be').id
+        partners = cls.env['res.partner'].with_context(**cls._test_context).create([{
+            'name': f'Partner_{x}',
+            'email': f'_test_partner_{x}@example.com',
+            'country_id': country_be_id,
+            'mobile': mobile_numbers[x]
+        } for x, mobile_number in enumerate(mobile_numbers)])
+        records = cls.env['mail.test.sms'].with_context(**cls._test_context).create([{
+            'name': f'MassSMSTest_{x}',
+            'customer_id': partner.id,
+            'phone_nbr': mobile_number
+        } for x, (mobile_number, partner) in enumerate(zip(mobile_numbers, partners))])
+
+        return partners, records
```

Comment:
now that we don't use Partner, we can just return records
```suggestion
        return records